models/account_abstraction/erc4337/arbitrum/account_abstraction_erc4337_arbitrum_v0_5_userops_basics.sql

```diff
@@ -1,21 +1,14 @@
 {{ config
-(
+(tags = ['dunesql'],
     alias = alias('v0_5_userops_basics'),
-    partition_by = ['block_time'],
+    partition_by = ['block_month'],
```

Comment:
partitions should be a larger aggregated value. this field will then be beneficial in downstream queries, whether in filters or group by's for quicker results

Reply:
date is most common, but could be a varchar/int/other type of column, depending on use case of spell